.github/workflows/nightly.yml

```diff
@@ -0,0 +1,65 @@
+name: julia nightly
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '1 0 1,15 * *'
+    
+jobs:
+  testMKLjl:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+#           - windows-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
```

Comment:
Same as https://github.com/JuliaMath/VML.jl/pull/31/files#r362649676

Reply:
yes, good point. I just pushed a second commit, because I forgot to add the changes to the test dependencies.